packages/server/server.js

```diff
@@ -149,15 +149,6 @@ const runServer = (
     fs.writeFile(image, req.body, err => console.log(err))
 
     res.status(200).send({ path: imagePath })
-    // const buff = Buffer.from(req.body, 'utf8').toString()
```

Comment:
@lebaphi - could you explain why these lines are commented out please?

Shouldn't we just delete them?

Reply:
I don't comment out them @IanMayo, I've seen it here before and I just removed them to clean up the code

Reply:
Sorry - I misinterpreted the PR.  Yes, I see you removed them :-)